src/sentry/static/sentry/app/redux/actions/shared.js

```diff
@@ -75,13 +75,25 @@ const acCreate = (resource, urlTemplate) => {
     dispatch(acCreateRequest(resource)(data));
 
     const url = urlTemplate.replace('{org}', org.slug);
-
-    return axios
-      .post(url, data)
-      .then(() => dispatch(acCreateSuccess(resource)(data)))
-      .catch((err) =>
-        dispatch(acCreateFailure(resource)(err.statusCode, err.statusText))
-      );
+    // use client that sentry is using
+    //
+    const api = new Client(); // TODO: use axios (must send same headers as Client does).
+    api.request(url, {
+      method: 'POST',
+      data,
+      success: () => {
+        dispatch(acCreateSuccess(resource)(data));
+      },
+      error: (err) => {
+        dispatch(acCreateFailure(resource)(err.statusCode, err.statusText));
+      },
+    });
+    // return axios
```

Comment:
Any reason to keep this commented out code?

Reply:
Yes. axios calls are currently not working with PUT and POST api calls, because we are missing an authentication header, which either Steinar or me have yet figured out how to write. I have written a ticket for this.

Reply:
Ok! I think it might be a good idea to add a comment with the JIRA ticket number, for context